css/index.css

```diff
@@ -82,16 +88,71 @@ h2 {
 }
 
 .connecting {
-    display: flex; /* sets the layout*/
+    /* sets the layout*/
+    display: flex;
     justify-content: center;
     align-items: center;
-    gap: 20px; /* space between each link*/
-    padding: 15px;
-    list-style-type: none; /* removes bullet points*/
+    gap: 20px;
+    /* space between each link*/
+    padding: 15px; 
+    /* removes bullet points*/
+    list-style-type: none;
+  
     text-align: center;
 }
 
-.connecting a {
-    text-decoration: none; /* Remove underline */
-    color: darkblue /* Set a color for icons/links */
+.connecting a { 
+    /* Remove underline */ 
+    text-decoration: none;
+   /* Set a color for icons/links */
+    color: darkblue
+       
+}
+
+/* to make changes for tablet devices */
```

Comment:
Can clearly see the differences, and you have documented it - so good.   AI reviewer wanted more comments, but I thought what you did is fine.